plugins/jenkins-backend/config.d.ts

```diff
@@ -22,6 +22,7 @@ export interface Config {
     baseUrl?: string;
     /**
      * Default instance username, can be specified on a named instance called "default"
+     * @visibility secret
```

Comment:
are usernames really sensitive?

Reply:
In some plugins it was defined as sensitive. I think we can leave it as default.